Formula/mariadb.rb

```diff
@@ -63,7 +63,7 @@ def install
     ]
 
     # Disable RocksDB becaus of build failure: https://jira.mariadb.org/browse/MDEV-13928
-    args << "-DPLUGIN_ROCKSDB=NO"
+    args << "-DPLUGIN_ROCKSDB=NO" if build.devel?
```

Comment:
I assume it's fixed in the next 10.3.x?

Reply:
I would expect so, the relevant fixes are in the 10.3 branch already.